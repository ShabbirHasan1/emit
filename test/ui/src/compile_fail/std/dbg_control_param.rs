fn main() {
    emit::dbg!(a: 42, "text");
}
