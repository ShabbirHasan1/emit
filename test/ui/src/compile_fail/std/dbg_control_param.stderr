error: specify the template string literal before any properties
 --> src/compile_fail/std/dbg_control_param.rs:2:5
  |
2 |     emit::dbg!(a: 42, "text");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `emit::dbg` (in Nightly builds, run with -Z macro-backtrace for more info)
